package-lock.json

```diff
@@ -3152,6 +3152,7 @@
       "version": "3.1.1",
       "resolved": "https://registry.npmjs.org/anymatch/-/anymatch-3.1.1.tgz",
       "integrity": "sha512-mM8522psRCqzV+6LhomX5wgp25YVibjh8Wj23I5RPkPppSVSjyKD2A2mBJmWGa+KN7f2D6LNh9jkBCeyLktzjg==",
+      "devOptional": true,
```

Comment:
Is this `"devOptional": true` scattered around ok?

Reply:
Yeah, I don't know about this - it came along with running `npm install` after updating the package, so I assume it's related, but hard to say 🤷 I'm hesitant to ignore it though as it was an automatic addition...